### PR TITLE
Add Forall construct highlight

### PR DESCRIPTION
**Pull request by pedro-ricardo**
Import forall highlight rule from [Atom fortran extension](https://github.com/dparkins/language-fortran).
**Before:**
before
**After:**
after

### CODE REVIEW

**Comment by krvajal** [comment]
Can you add this change to the Changelog in the  Unreleased section?

**Comment by krvajal** [comment]
Basically at the top of the list include a section like
```
## [Unreleased]

```
and put the stuff there.

**Comment by pedro-ricardo** [comment]
This should be a standard procedure for newest pull requests?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ All notable changes to this extension will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+* Add syntax highlight for 'forall' construct (#82)
```
Comment (krvajal) [inline CHANGELOG.md]: Please add like this, in the unreleased section
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,13 +5,17 @@ All notable changes to this extension will be documented in this file.
...
+* Syntax highlight for 'forall' construct (#82)
+
 ## [1.3.1] - 2018-06-26
 
 ### Fixed
 * Subroutine and function on symbol list were failing with more than one line arguments (#71)
 * Minor syntax highlighting issues (#62, #73)
 
-## Added 
+### Added
```
Comment (krvajal) [inline CHANGELOG.md]: Nice!

**Comment by krvajal** [comment]
Please feel free to merge this.

**Pull request merged**